catkit/hardware/iris_ao/segmented_dm_command.py

```diff
@@ -306,14 +315,15 @@ def to_command(self):
 
     def update_one_segment(self, segment_ind, ptt_tuple, add_to_current=True):
         """ Update the value of one segment by supplying the new command that will be added
-        to or will replace the current PTT tuple. To identify the segment to be changed, give
-        the *index* of that segment in the active segment list. This will be added
+        to or will replace the current PTT tuple on this segment only. To identify the segment to be changed, give
+        the *command index* (not its name like in the GUI) of that segment in the active segment list. This will be added
```

Comment:
typo. change to "it's"

Reply:
Are sure? This is not intended to be "it is", just "the/its name".